src/styles/card.scss

```diff
@@ -0,0 +1,61 @@
+.card {
+  @include space-between;
+  flex-direction: column;
+  width: 200px;
+  margin-bottom: 50px;
```

Comment:
BEM Blocks should not have outer styles (e.g. margin).

Reply:
fixed